README.md

```diff
@@ -169,6 +178,57 @@ generator = get_generator(config)
 dataset = generator.generate()
 ```
 
+## 🛠️ Supported tasks
+---
+![icon](https://github.com/Genaios/TextMachina/blob/main/assets/typewriter.png?raw=true) can generate datasets for MGT detection, attribution, boundary detection, and mixcase detection:
```

Comment:
Do these images load for you? I can't see them in the preview here.

Reply:
Not now. 

The urls point to the main branch, to be displayable once the PR is finished.